requirements/libraries.txt

```diff
@@ -37,14 +37,14 @@ requests_toolbelt==0.8.0
 sha3==0.2.1
 unipath>=1.1,<=2.0
 urllib3==1.25.2
-wagtail-autocomplete==0.6
```

Comment:
Maybe leave this commented out with a TODO about restoring it once a new release of wagtail-autocomplete comes out?

Reply:
Good idea. Done.